schutzbot/deploy.sh

```diff
@@ -20,8 +20,8 @@ function retry {
 
 # Variables for where to find osbuild-composer RPMs to test against
 DNF_REPO_BASEURL=http://osbuild-composer-repos.s3-website.us-east-2.amazonaws.com
-OSBUILD_COMMIT=376cbffd136bc4ba86fc7c63697fa5b88fe3acef             # release 62
-OSBUILD_COMPOSER_COMMIT=2d13050ba84a2bb30d9dd980639ee78354fe90f3    # release 59-dev (includes mock openid server fix)
+OSBUILD_COMMIT=fb5412931711be41fd19cdaf402f2ed89d1d60b8             # release 72
+OSBUILD_COMPOSER_COMMIT=587e0436681b74ecb359945066ae63afeba9085a    # release 68
```

Comment:
Any reason to not use the latest releases at this time?

Reply:
None, this PR is just old. Fixed in a force-push. ;)